CHANGES.md

```diff
@@ -1,5 +1,133 @@
+- Dagster Pipes and its integrations with Lambda, Kubernetes, and Databricks are no longer experimental.
+- The `Definitions` constructor will no longer raise errors when the provided definitions aren’t mutually resolve-able – e.g. when there are conflicting definitions with the same name, unsatisfied resource dependencies, etc. These errors will still be raised at code location load time. The new `Definitions.validate_loadable` static method also allows performing the validation steps that used to occur in constructor.
+- `AssetsDefinitions` object provided to a `Definitions` object will now be deduped by reference equality. That is, the following will now work:
+    
+    ```python
+    from dagster import asset, Definitions
+    
+    @asset
+    def my_asset(): ...
+      
+    defs = Definitions(assets=[my_asset, my_asset]) # Deduped into just one AssetsDefinition.
+    ```
+    
+- [dagster-embedded-elt] Adds translator options for dlt integration to override auto materialize policy, group name, owners, and tags
+- [dagster-sdf] Introducing the dagster-sdf integration for data modeling and transformations powered by sdf.
+- [dagster-dbt] Added a new `with_insights()` method which can be used to more easily attach Dagster+ Insights metrics to dbt executions: `dbt.cli(...).stream().with_insights()`
+
+### Bugfixes
+
+- Dagster now raises an error when an op yields an output corresponding to an unselected asset.
+- Fixed a bug that caused downstream ops within a graph-backed asset to be skipped when they were downstream of assets within the graph-backed assets that aren’t part of the selection for the current run.
+- Fixed a bug where code references did not work properly for self-hosted GitLab instances. Thanks, @cooperellidge!
+- [ui] When engine events with errors appear in run logs, their metadata entries are now rendered correctly.
+- [ui] The asset catalog greeting now uses your first name from your identity provider.
+- [ui] The create alert modal now links to the alerting documentation, and links to the documentation have been updated.
+- [ui] Fixed an issue introduced in the 1.7.13 release where some asset jobs were only displaying their ops in the Dagster UI instead of their assets.
+- Fixed an issue where terminating a run while it was using the Snowflake python connector would sometimes move it into a FAILURE state instead of a CANCELED state.
+- Fixed an issue where backfills would sometimes move into a FAILURE state instead of a CANCELED state when the backfill was canceled.
+
+### Breaking Changes
+
+- The experimental and deprecated `build_asset_with_blocking_check` has been removed. Use the `blocking` argument on `@asset_check` instead.
+- Users with `mypy` and `pydantic` 1 may now experience a “metaclass conflict” error when using `Config`. Previously this would occur when using pydantic 2.
+- `AutoMaterializeSensorDefinition` has been renamed `AutomationConditionSensorDefinition`.
```

Comment:
Please add:

```
- The deprecated methods of the `ComputeLogManager` have been removed.  Custom `ComputeLogManager` implementations must also implement the `CapturedLogManager` interface.  This will not affect any of the core implementations available in the core `dagster` package or the library packages.
```

Reply:
👍